### DISCUSSION TITLE
Auto re-path VRC Animator Play Audio source path when using Bone Proxy.

### DISCUSSION THREAD

**Discussion by dmcustom** [bdunderscore/modular-avatar#1178] [Ideas]
VRC Animator Play Audio just prompt that it can't find the source path of Audio Source anymore since the gameObject being moved to another path by Bone Proxy.

**Comment by bdunderscore** [comment]
Pending release, should be available in the prereleases
https://github.com/bdunderscore/modular-avatar/issues/935
